pkg/kudoctl/cmd/install/install_test.go

```diff
@@ -4,66 +4,21 @@ import (
 	"testing"
 )
 
-func TestInstallOperators(t *testing.T) {
-
-	// For test case #1
-	expectedNoArgumentErrors := []string{
-		"no argument provided",
-	}
-
-	// For test case #2
-	options := DefaultOptions
-	options.PackageVersion = "0.0"
-	installCmdPackageVersionArgs := []string{"one", "two"}
-	expectedPackageVersionFlagErrors := []string{
-		"--package-version not supported in multi operator install",
-	}
+func TestValidate(t *testing.T) {
 
 	tests := []struct {
-		args []string
-		err  []string
+		arg []string
+		err string
 	}{
-		{nil, expectedNoArgumentErrors},                                  // 1
-		{installCmdPackageVersionArgs, expectedPackageVersionFlagErrors}, // 2
+		{nil, "no argument provided, need name of the package or path to install"}, // 1
```

Comment:
not so happy with the testing here, shouldn't we test for the case I ran into e.g. if too many arguments are provided and zero arguments are provided? So not just the nil pointer?

```
$ go run cmd/kubectl-kudo/main.go install zk zookeeper kafka
Error: no argument provided, need name of the package or path to install
exit status 255
```